docs/topics/migrations.txt

```diff
@@ -27,6 +27,9 @@ and Django's handling of database schema:
 * :djadmin:`showmigrations`, which lists a project's migrations and their
   status.
 
+* :djadmin:`optimizemigration`, which optimizes the operations within a
+  migration.
```

Comment:
We could move this and a paragraph about `Optimizing migrations` to a separate PR e.g. `Refs #27844 -- Added optimizemigration to migrations topic guide`.

Reply:
I was thinking of doing this but then this PR will undo some of those changes (because they blend together)